client/blocks/simple-site-rename-form/index.jsx

```diff
@@ -14,12 +14,16 @@ import { connect } from 'react-redux';
 import Card from 'components/card';
 import FormButton from 'components/forms/form-button';
 import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+import FormInputValidation from 'components/forms/form-input-validation';
 import ConfirmationDialog from './dialog';
 import FormSectionHeading from 'components/forms/form-section-heading';
 import { requestSiteRename } from 'state/site-rename/actions';
 import { isRequestingSiteRename } from 'state/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
 
+const SUBDOMAIN_LENGTH_MINIMUM = 2;
```

Comment:
Wondering if the minimum length is correct here?
When calling a domain with less than for chars we display:
`Site name must be at least 4 characters.`
e.g. visit https://en.wordpress.com/typo/?subdomain=abc

Reply:
Hmm, that' odd - I can't actually remember where I've seen the 2 - 63 rule originally. 4+ seems reasonable though, especially given the guidance at https://en.wordpress.com/typo/?subdomain=abc